service/app/controllers/notification/NotificationController.java

```diff
@@ -17,22 +16,23 @@
  * @author manzarul
  *
  */
-public class NotificationController extends BaseController{
-	 Logger logger = LogManager.getLogger(NotificationController.class);
-	
-	public static final String NOTIFICATION = "notification";
-	
-	/**
-	   * This method will accept request for sending notification.
-	   * notification can be sent on email, sms or push on device
-	   * @return a CompletableFuture of success response
-	   */
-	  public CompletionStage<Result> sendNotification() {
-		logger.info("method call started for sendNotification " + request().body().asJson());
-		CompletionStage<Result> response = handleRequest(request(),null,NOTIFICATION);
-		logger.info("Method call end for sendNotification");
-	    return response;
-	  }
+public class NotificationController extends BaseController {
+    Logger logger = Logger.getLogger(NotificationController.class);
+
+    public static final String NOTIFICATION = "notification";
```

Comment:
Instead of defining this constant 2 times in actor as well as her , define this in some constant file